### PR TITLE
fix: Adjust sidebar banner to wrap bellow the title

### LINKED ISSUE #1607: Sidebar flex changes banner alignment issue
### Requirements

- [X] This is a bug report, and if not, please post to https://lemmy.ml/c/lemmy_support instead.
- [X] Please [check](https://github.com/LemmyNet/lemmy-ui/issues) to see if this issue already exists.
- [X] It's a single bug. Do not report multiple bugs in one issue.
- [X] It's a frontend issue, not a backend issue; Otherwise please create an issue on the [backend repo](https://github.com/LemmyNet/lemmy) instead.

### Summary

After upgrading to Lemmy 0.18.0 the sidebar banner and title are rendered in the same row.

Screenshot 2023-06-26 080031

A couple of changes to the component can fix the layout resulting in

Screenshot 2023-06-26 080147

Specifically:
https://github.com/LemmyNet/lemmy-ui/blob/46ab522143daedf23fb02476769650e0062fa392/src/shared/components/home/site-sidebar.tsx#L36C24-L36C24 < Add the `flex-wrap` class
 And https://github.com/LemmyNet/lemmy-ui/blob/46ab522143daedf23fb02476769650e0062fa392/src/shared/components/common/banner-icon-header.tsx#L19C68-L19C68 < Add `mx-auto` class to center the banner image

### Steps to Reproduce

1. Install Lemmy 0.18.0
2. Upload a banner image
3. Open front page

### Technical Details

This issue is visible on chromium desktop browsers

### Lemmy Instance Version

0.18.0

### Lemmy Instance URL

https://leddit.social/

### PR DESCRIPTION
**Pull request by oliverwhite19**
## Description
Fixes #1607 

Makes it so that the sidebar and banner don't appear in the same row.

### Before

Before

### After

Screenshot 2023-07-04 at 21 36 25

### CODE REVIEW

File: src/shared/components/home/site-sidebar.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ export class SiteSidebar extends Component<SiteSidebarProps, SiteSidebarState> {
       <div className="site-sidebar accordion">
         <section id="sidebarInfo" className="card border-secondary mb-3">
           <header
-            className="card-header d-flex align-items-center"
+            className="card-header d-flex align-items-center flex-wrap"
             id="sidebarInfoHeader"
           >
             {this.siteName()}
```
Comment (SleeplessOne1917) [inline src/shared/components/home/site-sidebar.tsx:39]: I'd put a bit of margin bottom on the site name when the banner is displaying, but other than that you're good.

**Comment by jsit** [comment]
I think rather than doing flex-wrap, these should probably just be on different rows to begin with. And I wonder if it would look better with the banner on top?

File: src/shared/components/home/site-sidebar.tsx (hunk 1 above)
Comment (oliverwhite19) [reply src/shared/components/home/site-sidebar.tsx:36]: I've conditionally added some margin on the header

**Comment by oliverwhite19** [comment]
> I think rather than doing flex-wrap, these should probably just be on different rows to begin with. And I wonder if it would look better with the banner on top?

I've set the elements on different rows and it does simplify things. I don't like the idea of putting the banner on top because then you move the collapse button up and down, which is bad UX imo

**Review by SleeplessOne1917 — APPROVED** [review]

File: src/shared/components/home/site-sidebar.tsx [hunk 2]
```diff
@@ -54,7 +51,7 @@ export class SiteSidebar extends Component<SiteSidebarProps, SiteSidebarState> {
 
   siteName() {
     return (
-      <>
+      <div className={!this.state.collapsed ? "mb-2" : undefined}>
```
Comment (dessalines) [inline src/shared/components/home/site-sidebar.tsx:54]: If the 2nd condition is empty, just do `!this.state.collapsed && "mb-2"` , or ever better, use the `classNames` util.

**Pull request merged**